src/_guides/language/common-prob.md

```diff
@@ -0,0 +1,473 @@
+---
+layout: guide
+title: "Sound Dart: Common Problems"
+description: "Common problems you may have when converting to strong mode and how to fix them."
+toc: false
+---
+
+If you are having problems converting your code to strong mode,
+this page can help. Be sure to also check out
+[Sound Dart](/guides/language/sound-dart) for an overview of what "sound
+Dart" means, and how strong mode contributes to making Dart a sound
+language.
+
+## Table of Contents
+
+<ul>
+<li><a href="#troubleshooting">Troubleshooting</a></li>
+<ul>
+<li><a href="#am-i-in-strong-mode">Am I really in strong mode?</a></li>
+<li><a href="#not-in-strong-mode">I'm not in strong mode and I think I should be</a></li>
+</ul>
+<li><a href="#common-errors">Common errors and warnings</a></li>
+<ul>
+<li><a href="#undefined-member">Undefined member</a></li>
+<li><a href="#invalid-method-override">Invalid method override</a></li>
+<li><a href="#unsound-implicit-downcast">Unsound implicit downcast</a></li>
+<li><a href="#missing-type-arguments">Missing type arguments</a></li>
+<li><a href="#assigning-mismatched-types">Assigning mismatched types</a></li>
+<li><a href="#constructor-initialization-list">Constructor initialization list super() call</a></li>
+</ul>
+<li><a href="#other-resources">Other resources</a></li>
+</ul>
+
+## Troubleshooting
+
+<a name="am-i-in-strong-mode"></a>
+### Am I really in strong mode?
+
+If you're not seeing strong mode errors or warnings,
+make sure that you are in strong mode.
```

Comment:
GLOBAL:
you are in -> you're using

(I am always in strong mode! Also, I am always in literal mode.)